README.md

```diff
@@ -777,37 +777,42 @@ for signing. The bot calls the script with the two arguments:
-architecture to a job (`ADDITIONAL_SBATCH_PARAMETERS`).
+Each entry in the `node_type_map` dictionary describes a build node type. The key is a (descriptive) name for this build node, and its value is a dictionary containing the following build node properties as key-value pairs:
+  - `os`: its operating system (os)
+  - `cpu_subdir`: its CPU architecture
+  - `slurm_params`: the SLURM parameters that need to be passed to submit jobs to it
+  - `repo_targets`: supported repository targets for this node type
+  - `accel` (optional): which accelerators this node has
+All values are strings, except repo_targets, which is a list of strings. Repository targets listed in `repo_target` should correspond to the repository IDs as defined in the `repos.cfg` file in the `repos_cfg_dir` (see below).
 
-Note, if you do not have to specify additional parameters to `sbatch` to request a compute node with a specific microarchitecture, you can just write something like:
+Note that the Slurm parameters should typically be chosen such that a single type of node (with one specific type of CPU and one specific type of GPU) should be allocated.
 
-```ini
-arch_target_map = { "linux/x86_64/generic": "" }
-```
+To command the bot to build on the `cpu_zen2` node type above, one would give the command `bot:build on:arch=zen2 ...`. To command the bot to build on the `gpu_h100` node type, one would give the command `bot:build on:arch=zen4,accel=nvidia/cc90 ...`.
 
-#### `[repo_targets]` section
+For a native build (i.e. building for `zen2` on a `zen2` node), one can pass `bot:build on:arch=zen2 for:arch=x86_64/amd/zen2`, or use the short-hand `bot:build for:arch=x86_64/amd/zen2` (i.e. omitting the `on` argument implies a native build). This will trigger a build on the `cpu_zen2` node type (as configured above) and prepare a configuration file in the job directory that instructs to build for a `zen2` CPU architecture.
 
-The `[repo_targets]` section defines for which repositories and architectures the bot can run a job.
-Repositories are referenced by IDs (or `repo_id`). Architectures are identified
-by `OS/SUBDIR` which correspond to settings in the `arch_target_map`.
+For cross-compiling GPU code for Nvidia Compute Capabiltiy 8.0 (and a `zen2` CPU architecture), one would instruct the bot with `bot:build on:arch=zen2 for:arch=x86_64/amd/zen2,accel=nvidia/cc80`. This will trigger a build on the `cpu_zen2` node type (as configured above) and prepare a configuration file in the job directory that instructs to build for a `zen2` CPU architecture with an `nvidia/cc80` GPU architecture.
 
-```ini
-repo_target_map = {
-    "OS_SUBDIR_1": ["REPO_ID_1_1","REPO_ID_1_2"],
-    "OS_SUBDIR_2": ["REPO_ID_2_1","REPO_ID_2_2"] }
-```
 
-For each `OS/SUBDIR` combination a list of available repository IDs can be
-provided.
+#### `[repo_targets]` section
+
+The `[repo_targets]` section defines for which repositories and architectures the bot can run a job.
```

Comment:
With the new code, this is mostly defined in the node map, right? And this section only still provides the CVMFS configuration for the repos defined in that map?